back/engines/commercial/analytics/app/services/analytics/query_runner_service.rb

```diff
@@ -43,6 +43,7 @@ def query_filters(results)
       @json_query[:filters].each do |dimension, columns|
         columns.each do |column, value|
           if [Array, String].include? value.class
+            value = convert_empty_to_null(value)
```

Comment:
What about something like:
```suggestion
            value = Array.wrap(value).map(&:blank?)
```

Reply:
How is this change related to the current PR? or is it fixing a bug?

Reply:
This is related to the front-end. Probably should be in the other PR - the front end sends an empty array instead of null/nil so we specifically need to convert it